src/Search_Replace_Command.php

```diff
@@ -142,15 +142,14 @@ public function __invoke( $args, $assoc_args ) {
 		$this->regex_delimiter =  \WP_CLI\Utils\get_flag_value( $assoc_args, 'regex-delimiter', '/' );
```

Comment:
Maybe it would be safer to use `chr(1)` as the default regex delimiter instead. Very low chance of collision.

Reply:
I have never used `chr()`, can you please explain more details?

Did you mean following?
```
$this->regex_delimiter =  \WP_CLI\Utils\get_flag_value( $assoc_args, 'regex-delimiter', chr(1) );
```

Reply:
`chr()` let's you retrieve a specific ASCII character by code. Here is a [list of ASCII character codes](http://www.asciitable.com/).

The useful part is that it can return characters that you cannot easily type, so with something like `chr(1)`, the chances of the delimiter being included in the pattern itself are practically null, whereas a `/` can easily be part of the actual pattern.

And yes, what you wrote is how you would use it then.

Reply:
> The useful part is that it can return characters that you cannot easily type, so with something like chr(1), the chances of the delimiter being included in the pattern itself are practically null

Oh, you are right! It is clever...

Reply:
Yes, ingenious, can't think of a downside. Could just use a literal `"\x01"` instead, though not important.